src/components/Button/Button.styles.tsx

```
@@ -0,0 +1,38 @@
+    shouldForwardProp: (prop) =>
+        !['backgroundColor', 'buttonSize', 'borderRadius'].includes(
+            prop.toString(),
+        ),
+})<StyledButtonProps>(
+    ({
+        buttonSize,
+        backgroundColor,
+        borderRadius,
+        theme: { palette, spacing },
+    }) => ({
+        backgroundColor: ButtonBackgroundColorMap[backgroundColor!],
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)